javascript.js

```diff
@@ -0,0 +1,20 @@
+// set variavles
```

Comment:
Typo inside comment in word `variavles`. Possible corrections:  `variables` (72)